src/main/resources/application.conf

```diff
@@ -404,6 +404,11 @@ mantis {
     # on - missing state node will be redownloaded from a peer and block execution will be retried. This can repeat
     #      several times until block execution succeeds
     redownload-missing-state-nodes = on
+
+    # See: https://github.com/ethereum/go-ethereum/pull/1889
+    fast-sync-block-validation-k = 100
+    fast-sync-block-validation-n = 2048
+    fast-sync-block-validation-x = 50
```

Comment:
How did you choose this X value? In the PR (white paper) `X=24`

Reply:
Ahh, so you doubled it because we treat target/pivot differently?

Reply:
Yes, I was not sure how to interpret that sentence, but my intuition is that validating `pivot - 2x to pivot` is the same as validating `pivot - x to pivot + x`. (and validation after pivot happens anyway in regular sync).